src/net/dns.rs

```diff
@@ -417,6 +417,13 @@ static DNS_PRELOAD: LazyLock<HashMap<&'static str, Vec<IpAddr>>> = LazyLock::new
             "mail.systemli.org",
             vec![IpAddr::V4(Ipv4Addr::new(93, 190, 126, 36))],
         ),
+        (
```

Comment:
Btw, how was this code generated?
Also probably need to add more resolutions here, i just added what nslookup tells for me.
cc @link2xt